### DISCUSSION TITLE
Uyuni licit use question - licensing/SLES/Ubuntu

### DISCUSSION THREAD

**Discussion by ipanmeri** [uyuni-project/uyuni#8786] [General]
Hi all,
sorry to bother, but I can't get my head around this and I cannot find appropriate answer in the documentation.
Here are some questions, who might have the patience and kindness to answer will have my gratitude and appreciation.

1. While SuSE Manager is a product by SuSE to be used under license, Uyuni Server Manager is open software drive by community efforts - correct?

2. While Uyuni offers support for several platforms and OS, I focus on SLES and Ubuntu, having to manage those two OS.
2.1 Having a support subscription for SLES I can save SCC Credentials in Uyuni and use official SLES repositories to mirror on a local Uyuni server, in oder to manage/update SLES servers in my environment, which makes some kind of hybrid situation using non-licensed/free Uyuni with subscription repositories - correct?
2.2 What about Ubuntu? I can add Ubuntu channels to the Uyuni server but I am not sure which entity serves the repositories for Ubuntu to be mirrored locally. Who provides the Ubuntu repositories, also SCC SuSE?
2.3 I have noticed that while experimenting with Uyuni and registering the first couple of clients they get also registered in SCC SuSE.
While SLES clients for which there is a subscription in place which would make it licit use, I am unsure about the Ubuntu part - do I need a subscription with SCC SuSE also for these ubuntu clients?
I know that when using SuSE Manager I have to license each Ubuntu server, which makes it quite costly.

3. So if the Uyuni platform is open/free, with SLES subscriptions in place, can I add Ubuntu as channel not bound to SCC SuSE, can I download Ubuntu packages directly from Canonical?
Would this situation constitute licit/legal/correct use of Uyuni?

Yes, I would use Uyuni in a professional environment. Total of <100 servers, most virtual.

Thank you very much, whoever has the patience and kindness to answer.
Kind regards

**Comment by mmmmmmpc** [comment]
Dear `ipanmeri` let me reply to your questions.
1.- SUSE Manager is a product by SUSE that is sold on a subscription basis. These subscriptions provide you support, certifications, the possibility to use professional services and more. Uyuni is the upstream project from which SUSE Manager is derived, and it is a community effort.
2.- You have to understand that by using Uyuni, you are covered by community voluntary efforts and therefore not under any support contract. 
2.1.- In this case you can only claim support for SLES neither for Uyuni nor anything that you do on SLES from your Uyuni server. Neither Uyuni, nor SUSE Manager have licenses as both are Open source, however, you can acquire subscriptions for SUSE if you want to be professionally supported. By choosing Uyuni you choose to work only with the community,
2.2.- Ubuntu repositories are provided by the Ubuntu community and even when both Uyuni and SUSE Manager can handle them, you have to configure. For SUSE Manager we provide the repository configuration via SCC. You are not entitled to it unless you have a valid SUSE Manager subscriptions.
2.3.- To use the repository information for Uyuni from SCC you need SUSE Manager subscriptions. If you do not want to pay for the SUSE Manager subscriptions, you can keep using Uyuni but that means you will have to configure Ubuntu channels manually.
3.- SLES subscriptions do not entitle you to use SUSE Manager or the services in SCC intended to be used with SUSE Manager. You can use Uyuni downloading the Ubuntu packages from the mirrors available worldwide without SCC. Using Uyuni to manage SLES is not a supported case of SLES.
You can always subscribe SLES systems to SCC directly or use RMT with comes included with SLES to manage its repositories: https://documentation.suse.com/sles/15-SP4/single-html/SLES-rmt/index.html

My recommendation is that you check with your Account Representative or SUSE Partner on the cost of SUSE Manager to handle all servers with the support from SUSE, just the same way you get it for SLES.

Happy to help and hope you keep enjoying SLES, Uyuni and SUSE Manager.
